src/lib/consensus/intf.ml

```diff
@@ -59,21 +67,26 @@ module type Blockchain_state = sig
     , Frozen_ledger_hash.var
     , Mina_transaction_logic.Zkapp_command_logic.Local_state.Checked.t
     , Block_time.Checked.t
-    , Body_reference.var )
+    , Body_reference.var
+    , Currency.Amount.Signed.var
+    , Pending_coinbase.Stack.var
+    , Fee_excess.var
+    , Sok_message.Digest.Checked.t )
     Poly.t
 
   val staged_ledger_hash :
-    ('staged_ledger_hash, _, _, _, _) Poly.t -> 'staged_ledger_hash
+    ('staged_ledger_hash, _, _, _, _, _, _, _, _) Poly.t -> 'staged_ledger_hash
 
   val snarked_ledger_hash :
-    (_, 'frozen_ledger_hash, _, _, _) Poly.t -> 'frozen_ledger_hash
+    (_, 'frozen_ledger_hash, _, _, _, _, _, _, _) Poly.t -> 'frozen_ledger_hash
 
   val genesis_ledger_hash :
-    (_, 'frozen_ledger_hash, _, _, _) Poly.t -> 'frozen_ledger_hash
+    (_, 'frozen_ledger_hash, _, _, _, _, _, _, _) Poly.t -> 'frozen_ledger_hash
 
-  val timestamp : (_, _, _, 'time, _) Poly.t -> 'time
+  val timestamp : (_, _, _, 'time, _, _, _, _, _) Poly.t -> 'time
 
-  val body_reference : (_, _, _, _, 'body_reference) Poly.t -> 'body_reference
+  val body_reference :
```

Comment:
nit: the type parameter above is `body_ref`, rename either this function or the type parameter so parameter and function have the same name